src/App.jsx

```diff
@@ -8,10 +8,16 @@ import { useState } from "react"
 import Search from "./components/Search"
 import SingleArticle from "./components/SingleArticle"
 import { UserProvider } from "./contexts/User"
+import TopicDropdown from "./components/TopicDropdown"
+import TopicPage from "./components/TopicPage"
 
 function App() {
 const [searchTerm, setSearchTerm] = useState("")
 
+const topics = [     { slug: "coding", description: "Code is love, code is life" },
```

Comment:
The topics should come from the api rather than be hard coded - this will mean if any new topics are added, we'll always have the most up to date list. We also want to avoid having that state in the top level of the app as that will cause the whole app to re-render every time it is fetched